### ISSUE TITLE
Latest Release Looks like a Trojan?

### ISSUE THREAD

**Issue by Mike-the-Monsta** [ZeroFox5866/nucleuscoop#30]
When attempting to download the rar of the latest release, windows defender throw the following notifications
![WIndows Defender](https://user-images.githubusercontent.com/24611770/110498520-6832bb80-80c5-11eb-8830-7103c15ee68a.PNG)

Is this safe to download?

**Comment by Theplainface** [comment]
Yeah same here. However it does say Trojan Script which means it is acting like a trojan so it might not actually be one. As for the backdoor that's something I am very weary of. It allows for updates but at the same time the stealing of information if installed. Please address this or I will have to ask the website to take it down under the Privacy and Security site policy as listed here https://docs.github.com/en/github/site-policy/github-marketplace-developer-agreement#5-representations-warranties-and-covenants. 
Thank You,
Hope this clears soon, I want to play halo split screen soon!

**Comment by Javrel** [comment]
same here, flagged as virus by windows

**Comment by IFireflyl** [comment]
Most likely the item being flagged is NucleusCoop\utils\GoldbergEmu\experimental_steamclient\steamclient_loader.exe. If I'm not mistaken, virtually any emulation is seen by anti-virus programs as bad. This is most likely a false positive.

**Comment by VladislavMm** [comment]
![image](https://user-images.githubusercontent.com/53298335/113943936-56e7e680-980c-11eb-833b-c07ce8181828.png)
![image](https://user-images.githubusercontent.com/53298335/113943953-623b1200-980c-11eb-96c8-38123d0110d9.png)
Checked through VirusTotal. And the most interesting thing is that in the past versions there was not such an abundance of "viruses" files like the same with what is it connected?

**Comment by VladislavMm** [comment]
![image](https://user-images.githubusercontent.com/53298335/113944493-45530e80-980d-11eb-91c8-b322a3ce7de1.png)
![image](https://user-images.githubusercontent.com/53298335/113944527-51d76700-980d-11eb-8aba-c1b1eca2353d.png)
![image](https://user-images.githubusercontent.com/53298335/113944543-5d2a9280-980d-11eb-9cdd-662283c53f07.png)
For example, past versions

**Comment by WizzardMaker** [comment]
Those are all automated flags, because this project uses methods, which are often misused in viruses and the likes (code injection, etc) 

> Yeah same here. However it does say Trojan Script which means it is acting like a trojan so it might not actually be one. As for the backdoor that's something I am very weary of. It allows for updates but at the same time the stealing of information if installed. Please address this or I will have to ask the website to take it down under the Privacy and Security site policy as listed here https://docs.github.com/en/github/site-policy/github-marketplace-developer-agreement#5-representations-warranties-and-covenants.

This project is not a github marketplace application. Those terms simply do not apply.

**Comment by steven-slezak** [comment]
@WizzardMaker so you're saying this is being flagged because the methods used to build this mod are similar to ones in Trojans and that's why it's being flagged? You believe this is still safe to attempt to download/install? Thank you

**Comment by WizzardMaker** [comment]
As far as the code goes presented on this github repo, yes, this program should contain no trojan

There is always a chance for false positives, especially when it comes to code that modifies other processes

Anti virus software rather flags safe programs, than ignore potential viruses

**Comment by steven-slezak** [comment]
I feel better and I've learned something new. Thanks @WizzardMaker.
